src/interaction-handlers/joinButton.ts

```diff
@@ -1,29 +1,26 @@
 import type { ButtonInteraction } from 'discord.js';
-import type { Player } from '#game/Game';
 import { InteractionHandler, InteractionHandlerTypes } from '@sapphire/framework';
 import { ApplyOptions } from '@sapphire/decorators';
 import { CustomIds, EmbedColor } from '#types/Enums';
 import { createEmbed } from '#root/lib/utils/responses';
 
-@ApplyOptions<InteractionHandler.Options>({ interactionHandlerType: InteractionHandlerTypes.Button, enabled: false })
+@ApplyOptions<InteractionHandler.Options>({ interactionHandlerType: InteractionHandlerTypes.Button, enabled: true })
```

Comment:
Make sure you remove `enabled: true` when the pr is undrafted since it's default like that